rapid7_insightvm/plugin.spec.yaml

```
@@ -2,9 +2,9 @@ plugin_spec_version: v2
 extension: plugin
 products: [insightconnect]
 name: rapid7_insightvm
-title: Rapid7 InsightVM
-description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin can get top remediations, scan results and start scans
-version: 4.9.0
+title: Rapid7 InsightVM Console
+description: InsightVM is a powerful vulnerability management tool which finds, prioritizes, and remediates vulnerabilities. This plugin uses an orchestrator to get top remediations, scan results and start scans
+version: 5.0.0
```

Perhaps, checking with Mike:
```suggestion
version: 4.9.1
```